spec/session_ssl_spec.rb

```diff
@@ -258,6 +257,16 @@
     expect(body.request_method).to be == "GET"
   end
 
+  it "should work when ssl_cert is supplied" do
+    @session.insecure = nil
+    @session.ssl_cert = 'spec/certs/keystore.p12'
+    @sessions.ssl_cert_type = "p12"
```

Comment:
`@sessions` should probably be `@session`? And if it isn't does this test pass?

Reply:
Yeah, my bad